.github/workflows/main.yml

```diff
@@ -0,0 +1,13 @@
+name: CI
+
+on: [pull_request, workflow_dispatch]
+
+jobs:
+  main_ci:
+    runs-on: ubuntu-latest
```

Comment:
ubuntu-20.04

Reply:
The industrial action runs the docker instance etc. on 22.04 aka latest as described in the documentation. Only the container that is started inside runs 20.04. This is kind of weird I know, but they do it this way. And we have no benefit of running the host system at an older Ubuntu Version.